app/web/features/search/SearchResultsList.tsx

```diff
@@ -147,34 +162,33 @@ export default function SearchResultsList({
             breakpoint="sm"
             className={classes.scroller}
             isFetching={isLoading}
-            fetchNext={fetchNextPage}
+            // fetchNext={fetchNextPage} // TODO: disabled for now (until pagination)
```

Comment:
let me disable this (which honestly I doubt that any user hardy uses it) and map is fixed :D